db/schema.rb

```diff
@@ -159,7 +167,7 @@
     t.string   "availability"
     t.string   "isbn"
     t.string   "language"
-    t.string   "physical_description"
+    t.text     "physical_description"
```

Comment:
But here you changed to text?  How does the text-string conversion work?

Reply:
@daryanypl in Rails, the Schema is useful for creating a database, and migrations are used for changing an existing database (ie converting a field type from string to text).  So changes to the schema have no affect on existing databases.  I made this code change to revert a change to this line on a previous commit.  So this PR as a whole doesn't make a change to this file.